api/src/main/java/javax/enterprise/inject/spi/builder/Builders.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * JBoss, Home of Professional Open Source
+ * Copyright 2016, Red Hat, Inc., and individual contributors
+ * by the @authors tag. See the copyright.txt in the distribution for a
+ * full listing of individual contributors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package javax.enterprise.inject.spi.builder;
+
+import java.util.ServiceLoader;
+
+/**
+ * Provide access to CDI meta-data builders.
+ *
+ * @author Antoine Sabot-Durand
+ * @since 2.0
+ */
+public final class Builders {
+
+    protected static volatile BuildersProvider buildersProvider = null;
+
+    public static <T> AnnotatedTypeBuilder<T> annotatedType(Class<T> type) {
+        return provider().getAnnotatedTypeBuilder(type);
+    }
+
+    public static AnnotatedTypeBuilder<?> annotatedType() {
```

Comment:
I think the name of this method could be confusing. It's true that `annotatedTypeBuilder` is longer but perhaps little bit more clear?